.github/workflows/test-digitalocean-install.yaml

```diff
@@ -7,7 +7,7 @@
 #
 name: e2e - DigitalOcean (install)
 
-on: push
+on: [push, workflow_dispatch]
```

Comment:
Note that most other workflows we have (I'd use posthog/ repo as a canonical example) uses pull_request trigger.

`workflow_dispatch` feels like YAGNI but w/e :)

Reply:
I've added `workflow_dispatch` as requested by @tiina303 [here](https://github.com/PostHog/charts-clickhouse/pull/200#discussion_r755463606). 

Regarding `pull_request` VS `push` we can revisit this if we think CI checks will run too often.